app/views/reactivate_account/index.html.erb

```diff
@@ -29,13 +29,14 @@
   <%= t('instructions.account.reactivate.explanation') %>
 </p>
 
-<div class="scale-down">
+<figure class="margin-0 scale-down" aria-label="<%= t('users.personal_key.accessible_labels.preview') %>">
```

Comment:
Not within the scope of this ticket, but if this were to be made a `<figure>` element, would it make sense for the personal key partial to be made into a `<figcaption>` element instead of a typical `<div>` element?

Reply:
I was on the fence about making this change. Visually, it's meant to look like a sort of embedded preview example, and I thought it'd be useful to indicate that somehow rather than just skipping straight to the first bit of content in the preview. I think including a `<figcaption>` could make sense if the visible preview had some sort of caption. In this implementation, the `aria-label` serves the purpose captioning the figure, and the personal key partial is the contents of the figure example preview.